website/docs/features/github-action.mdx

```diff
@@ -72,26 +72,26 @@ You can also check out the quickstart guide for GitHub actions in the GitHub doc
 ### Multiple step action
 ```yaml
 - name: compile contracts
-    uses: ecadlabs/taqueria-github-action@v0.2.2
+    uses: ecadlabs/taqueria-github-action@v0.8.0
```

Comment:
The way that we specify the command and plugin to be invoked looks off. @GImbrailo would you mind confirming please?